### PR TITLE
Adding vexchange-uniswap delta + other minor updates

### PR DESCRIPTION
**Pull request by leodsoft**
Additional updates: some references to “whitepaper” to qualify as “Uniswap whitepaper”; updated references to chainid as “1” ==> left the v1 documentation untouched.

### CODE REVIEW

**Comment by leodsoft** [comment]
Guys - see first draft of reference on Vexchange v2 contracts; feel free to recommend different style etc, or need for additional content.

As mentioned - I also updated some details I noted around the migration, such as some references to the chained and DAI ==> VTHO addresses; you can see these as separate commits in the PR if needed.

I haven't attempted to run the compile / rendering step to verify the structure is correct yet - I attempted to prepare this page as an inserted entry in the existing set of topics, so it might need some massaging for the correct structure syntax.

File: src/pages/docs/v2/01-protocol-overview/02-vexchange-uniswap-delta.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+event DefaultPlatformFeeChanged(uint oldDefaultPlatformFee, uint newDefaultPlatformFee);
+event DefaultRecovererChanged(address oldDefaultRecoverer, address newDefaultRecoverer);
+```
+
+### Factory Construction
+The VexchangeV2Factory constructor has a new signature, but a functionally simple change. It adds parameters for initializing the default fee parameters, and the default recoverer account.
+
+```solidity
+constructor( uint _defaultSwapFee, uint _defaultPlatformFee, address _defaultRecoverer ) public
+
```
Comment (OliverNChalk) [inline src/pages/docs/v2/01-protocol-overview/02-vexchange-uniswap-delta.md:65]: Add `;` to line end and drop new line from codeblock?

File: src/pages/docs/v2/08-SDK/08-other-exports.md [hunk 2]
```diff
@@ -26,11 +26,8 @@ A union type comprised of all types that can be cast to a JSBI instance.
 ```typescript
 import { ChainId } from '@uniswap/sdk'
```
Comment (OliverNChalk) [inline src/pages/docs/v2/08-SDK/08-other-exports.md:27]: Does this need to be updated to talk about a `@vexchange/sdk` package? @KennethAshley

**Review by OliverNChalk — APPROVED** [review]
Looks good to me, approved. Just one comment on the delta and a question for Kenneth

File: src/pages/docs/v2/08-SDK/08-other-exports.md (hunk 2 above)
Comment (KennethAshley) [reply src/pages/docs/v2/08-SDK/08-other-exports.md:27]: Yep. Ive created our own package. Its “vexchange-sdk”.

Comment (KennethAshley) [reply src/pages/docs/v2/08-SDK/08-other-exports.md:27]: I'll update

**Review by KennethAshley — APPROVED** [review]
Looks good to me as well. I'll go through and update all the uniswap packages to our vexchange packages

**Pull request merged**